package.json

```diff
@@ -1,22 +1,14 @@
 {
-  "name": "kappa-workshop",
-  "description": "P2P collaboration workshop",
+  "name": "p2p-kappa-chat",
+  "description": "PxP. P2P kappa core based chat for pirates",
```

Comment:
Are you leaving the "pirates" in the description? ⚓️

Reply:
haha,  I think it is funny ay ay

Reply:
🏴‍☠️ 🦜